edpm_ansible/roles/edpm_nftables/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
I think we should remove that file - it's actually not used at all in EDPM world, and may even lead to unwanted behavior. It was a compatibility layer in tripleo world..

Reply:
I agree with you but for its use, but I'd prefer to comment the include_tasks within rather than remove the entire file.

Reply:
ok. Maybe a "fail" with some indication about the usage? but otherwise OK.